.github/workflows/push-event.yml

```diff
@@ -51,6 +51,7 @@ jobs:
           next_patch=$((patch + 1))
           next_patch_version_name="$major.$minor.$next_patch"
           echo "$next_patch_version_name" > versionName.txt
+          echo "NEW_VERSION_NAME=$next_patch_version_name" >> $GITHUB_OUTPUT
```

Comment:
Might want to revert this. We can use the last name. This would eventually break the next steps that use the last name